File: eval/fixtures/legacy-directory-backed-asmx-commerce/src/Legacy/Orders/OrderFulfillmentManager.cs
namespace Acme.Legacy.Orders;

public class OrderFulfillmentManager
{
  private readonly OrderFulfillmentRepository _repository = new OrderFulfillmentRepository();

  public string RepriceBacklog(string orderId)
  {
    return _repository.WriteFulfillmentReprice(orderId);
  }
}

File: eval/fixtures/legacy-directory-backed-jaxws-commerce/src/main/java/com/acme/legacy/orders/OrderFulfillmentManager.java
package com.acme.legacy.orders;

public class OrderFulfillmentManager {
  private final OrderFulfillmentRepository orderFulfillmentRepository = new OrderFulfillmentRepository();

  public FulfillmentRepriceResult repriceBacklog(String orderId) {
    return orderFulfillmentRepository.writeFulfillmentReprice(orderId);
  }
}
